src/cls/rgw/cls_rgw_ops.h

```diff
@@ -1122,21 +1146,35 @@ struct cls_rgw_lc_list_entries_op {
 WRITE_CLASS_ENCODER(cls_rgw_lc_list_entries_op)
 
 struct cls_rgw_lc_list_entries_ret {
-  map<string, int> entries;
+  vector<cls_rgw_lc_entry> entries;
   bool is_truncated{false};
 
   cls_rgw_lc_list_entries_ret() {}
 
   void encode(bufferlist& bl) const {
-    ENCODE_START(2, 1, bl);
+    ENCODE_START(3, 3, bl);
```

Comment:
and here

Reply:
@yaozongyou I know it's not the most efficient, but it's a rare enough request to allow it, and potentially could help with upgrade tests